### PR TITLE
Add a note in the PR template reminding to follow NuGet guidelines 

### PR DESCRIPTION
**Pull request by Nigusu-Allehu**
# Bug

Fixes: 

## Description
This PR tries to make sure our guidelines are more visible by adding them to the PR template. I added the guidelines as a note in a comment section of the template. However, we could also have them as checklists as proposed in my comment. This will make sure the owner of the PR at least is aware of the guidelines.

## PR Checklist

- [x] Meaningful title, helpful description and a linked NuGet/Home issue
- [ ] Added tests
- [ ] Link to an issue or pull request to update docs if this PR changes settings, environment variables, new feature, etc.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -14,3 +14,10 @@ Fixes:
 - [ ] Meaningful title, helpful description and a linked NuGet/Home issue
 - [ ] Added tests
 - [ ] Link to an issue or pull request to update docs if this PR changes settings, environment variables, new feature, etc.
+
+<!--
+Note please make sure you follow the following guidlines.
+https://github.com/NuGet/NuGet.Client/blob/dev/docs/feature-guide.md
+https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md
+https://github.com/NuGet/NuGet.Client/blob/dev/docs/localizability.md
+-->
```
Comment (Nigusu-Allehu) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion

- [ ] (Follows feature guidline)[https://github.com/NuGet/NuGet.Client/blob/dev/docs/feature-guide.md] 
- [ ] (Follows coding guidline)[https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md] 
- [ ] (Follows localizability guidline)[https://github.com/NuGet/NuGet.Client/blob/dev/docs/localizability.md] 
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -14,3 +14,10 @@ Fixes:
 - [ ] Meaningful title, helpful description and a linked NuGet/Home issue
 - [ ] Added tests
 - [ ] Link to an issue or pull request to update docs if this PR changes settings, environment variables, new feature, etc.
+
+<!--
+Note please make sure you follow the following guidlines.
```
Comment (jeffkl) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
Note please make sure you follow the following guidelines.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jeffkl) [reply .github/PULL_REQUEST_TEMPLATE.md]: I'm not a fan of adding more checkboxes

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -14,3 +14,10 @@ Fixes:
 - [ ] Meaningful title, helpful description and a linked NuGet/Home issue
 - [ ] Added tests
 - [ ] Link to an issue or pull request to update docs if this PR changes settings, environment variables, new feature, etc.
+
+<!--
+Note: please make sure you follow the following guidelines.
+https://github.com/NuGet/NuGet.Client/blob/dev/docs/feature-guide.md
+https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md
```
Comment (nkolev92) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: Should the contributing doc be refencing all of these instead?
Feels like it's the landing point for development, so I'd everything to be linked there and kept up to date. It's also something I've seen in other repos.

Comment (donnie-msft) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: +1 - just need a comment linking to our contributing guide.
But if you are going to try to list/maintain them here, I'd like the UI guidelines to be included.

**Comment by donnie-msft** [comment]
> This will make sure the owner of the PR at least is aware of the guidelines.

Isn't it already common/best-practice to read contributing guides before submitting PRs?  I'm curious what lead to this... was something missed by a contributor?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (Nigusu-Allehu) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: I think it is great points raised here. I guess https://github.com/NuGet/NuGet.Client/blob/dev/CONTRIBUTING.md#resources-for-nugetclient-development should be enough then

**Comment by Nigusu-Allehu** [comment]
> > This will make sure the owner of the PR at least is aware of the guidelines.
> 
> Isn't it already common/best-practice to read contributing guides before submitting PRs? I'm curious what lead to this... was something missed by a contributor?

I created this because when I was working on SDKAnalysisLevel introduction to NuGet specifically restore, the guideline regarding introducing a new feature would have been really useful. I was thinking maybe it would be useful to others opening new PRs to be aware these guidelines would be timesaving. However, if one follows all the links in the Contributing.md, they should be aware of these guidelines as well. As a result, I am closing this PR.

**Pull request closed without merging**